applications/DEMApplication/DEM_application_variables.h

```diff
@@ -131,6 +131,14 @@ namespace Kratos
   KRATOS_DEFINE_APPLICATION_VARIABLE(DEM_APPLICATION, int, IF_BOUNDARY_ELEMENT)
   KRATOS_DEFINE_APPLICATION_VARIABLE(DEM_APPLICATION, Vector, IF_BOUNDARY_FACE)
   KRATOS_DEFINE_APPLICATION_VARIABLE(DEM_APPLICATION, DenseVector<int>, PARTICLE_CONTACT_FAILURE_ID)
+  KRATOS_DEFINE_APPLICATION_VARIABLE(DEM_APPLICATION, double, BEAM_CROSS_SECTION)
```

Comment:
I would use the same variables as in StructuralMechanics to be consistent

Reply:
Done for those that coincide. Thanks for the advice.